src/Management/test/Endpoint.Test/Metrics/Observer/EventCounterListenerTest.cs

```diff
@@ -62,9 +62,11 @@ public void EventCounterListenerGetsMetricsTest()
         }
     }
 
-    [Fact(Skip = "See https://github.com/SteeltoeOSS/Steeltoe/issues/985")]
+    [Fact]
```

Comment:
Should this PR auto-close #985?

Reply:
I was going to see if we observe any more failures after this resolution.

Reply:
Ok